src/resources/js/components/ListPresentations.vue

```diff
@@ -0,0 +1,110 @@
+<template>
+  <v-container>
+    <v-alert
+    v-model="message.show"
+    :class="message.type"
+    dismissible
+    >
+      {{ message.content }}
+    </v-alert>
+
+      <v-row align="center">
+      <v-card
+        class="mx-auto"
+        max-width="400"
+        tile
+      >
+        <v-list
+          shaped="shaped"
+        >
+          <v-subheader>Presentations</v-subheader>
+          <v-list-item-group v-model="presentations" color="primary">
+            <v-list-item
+              v-for="(presentation) in presentations.data"
+              :key="presentation.id"
+              @click="goToPresentation(presentation.id)"
```

Comment:
You should add a button instead of a click on the list item, so we can implement more buttons for a presentation, for example a button to leave the presentation directly from the list of presentations

Reply:
ok, added in [395b2](https://github.com/kayoumido/HEIGVD-PRO-A-11/pull/115/commits/395b23cf1ddd1d6e077eb19f0644c6cce23a4b09)